### PR TITLE
JBPM-5266: Jobs: Type validation fails during new job creation

### PR DESCRIPTION
**Pull request by nmirasch**
@mswiderski can you take a look please?  thanks!

### CODE REVIEW

File: kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/ExecutorServiceBase.java [hunk 1]
```diff
@@ -73,6 +73,11 @@ public String scheduleRequest(String containerId, String payload, String marshal
             jobRequest = marshallerHelper.unmarshal(payload, marshallingType, JobRequestInstance.class);
         }
         Long requestId = null;
+
+        if(!validateCommand(jobRequest.getCommand())){
+            throw new IllegalArgumentException("Invalid command type");
```
Comment (mswiderski) [inline kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/ExecutorServiceBase.java]: can we add class (jobRequest.getCommand()) that was not found to provide direct feedback?

**Comment by mswiderski** [comment]
@nmirasch just one tiny comment and besides that it looks ok

File: kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/ExecutorServiceBase.java (hunk 1 above)
Comment (psiroky) [inline kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/ExecutorServiceBase.java]: +1 to add the class name into the exception message

**Comment by sutaakar** [comment]
@nmirasch I have an idea about way how to change testScheduleAndRequeueJob() test implementation to keep same coverage, please see https://github.com/sutaakar/droolsjbpm-integration/commit/d2e610427551aaabcaf99db4c401ac20d29a8d80 .
That way we can still reach job failure to check requeueRequest() functionality.

**Comment by nmirasch** [comment]
@mswiderski, @psiroky  added the class name to the exception message. @sutaakar changed the testScheduleAndRequeueJob following your suggestion. Can you review?

**Comment by sutaakar** [comment]
Looks good to me.

**Pull request merged**